### ISSUE TITLE
Unable to view group participants on Android

### ISSUE THREAD

**Issue by Julesssss** [Expensify/App#10642]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
- Create a group chat, or navigate to an existing one
- Tap the Group participant icons

## Expected Result:
- The list of participants should be displayed

## Actual Result:
- The 'Uh oh, an error occured'

![Screenshot_20220829-115558](https://user-images.githubusercontent.com/10736861/187180857-fe2cb693-2414-448f-aebb-1f6ff5a5c9ee.png) ![Screenshot_20220829-115638](https://user-images.githubusercontent.com/10736861/187180864-074294df-4af7-48b9-8cee-4bd642f257bb.png)

## Workaround:
I couldn't find a workaround for this, but it works for single participants

## Platform:

Where is this issue occurring?

- Web ❌ 
- iOS ❌ 
- Android ✅ 
- Desktop App ❌ 
- Mobile Web ❌ 

**Version Number:**  `1.1.92-0`
**Reproducible in staging?:** ✅ 
**Reproducible in production?:** -- I can't test this currently, as I would have to build a local Android prod build...
**Email or phone of affected tester (no customers):**  Android Pixel 6a
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:** 
**Issue reported by:** @Julesssss 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856

```
E/unknown:ReactNative: TypeError: Cannot read property 'displayName' of undefined
    
    This error is located at:
        in ReportParticipantsPage
        in withOnyx(ReportParticipantsPage)
        in Unknown
        in Unknown
        in StaticContainer
        in EnsureSingleNavigator
        in SceneView
        in RCTView
        in Unknown
        in RCTView
        in Unknown
        in RCTView
        in Unknown
        in CardSheet
        in RCTView
        in Unknown
        in AnimatedComponent
```

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Julesssss** [comment]
Hey @arielgreen, I'm not 100% this is reproducible for everyone yet so I'll ask in Slack too. 

For me this occurs even after logout. I thought that it might be related to the fake user, but it's not reproducible on iOS, with the same app version 😕

**Comment by peterdbarkerUK** [comment]
I couldn't reproduce
![image](https://user-images.githubusercontent.com/16888060/187184253-4111920e-9ca1-4331-a105-85f93106b414.png)

**Comment by peterdbarkerUK** [comment]
Could it be an internet connectivity issue?

**Comment by aimane-chnaif** [comment]
It's a duplicate of https://github.com/Expensify/App/issues/10492 and https://github.com/Expensify/App/pull/10554 fixes this crash and it should not happen from v1.1.93-0
So I think in this GH, need to confirm how the crashed users will look like on this page
According to this [PR](https://github.com/Expensify/App/pull/10554), it will display login (email) for displayName as a fallback

**Comment by marcochavezf** [comment]
> I couldn't reproduce ![image](https://user-images.githubusercontent.com/16888060/187184253-4111920e-9ca1-4331-a105-85f93106b414.png)

I can't reproduce it either

**Comment by Julesssss** [comment]
Confirmed that it is fixed as of `1.1.93-4`, closing
